app/src/main/java/com/keylesspalace/tusky/appstore/EventsHub.kt

```diff
@@ -1,20 +1,21 @@
 package com.keylesspalace.tusky.appstore
 
-import io.reactivex.rxjava3.core.Observable
-import io.reactivex.rxjava3.subjects.PublishSubject
+import android.util.Log
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableSharedFlow
 import javax.inject.Inject
 import javax.inject.Singleton
 
 interface Event
-interface Dispatchable : Event
```

Comment:
Not sure why there were two interfaces? Seems unnecessay? @charlag?

Reply:
I think the idea was to mark only certain events as the ones that can be sent but it makes little sense anyway